src/main/java/org/mongojack/internal/util/DocumentSerializationUtils.java

```diff
@@ -392,52 +393,7 @@ public static Document serializeDBUpdate(
     public static Document serializeDBUpdateAsDocument(
             Map<String, Map<String, UpdateOperationValue>> update,
             ObjectMapper objectMapper, JavaType javaType) {
```

Comment:
This method is an exact copy of `serializeDBUpdate(update, objectMapper, javaType)` in same class, and has no usages in mongojack (neither main nor test). As it is public I did not remove it, but just made it call the other method.

Reply:
It isn't in a released version yet, so it should be removed... My fault. When I was trying to get Document methods working my first run through left it in the original SerializationUtils as a separate method. Later I realized that wouldn't worked and copied the old the new, replacing DBObject with Document... We obviously don't need two methods that do the same thing though.